packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -225,11 +213,12 @@ describe("single-choice question", () => {
             renderQuestion(q, apiOptions);
 
             // Act
-            userEvent.type(screen.getAllByRole("radio")[0], "{arrowup}");
-            userEvent.type(screen.getAllByRole("radio")[3], "{arrowup}");
+            userEvent.tab();
+            userEvent.tab();
+            userEvent.tab();
```

Comment:
🤔  So this changes the behaviour to remove arrow key navigation and replace it with tabbing.  I suspect that's a more common navigation pattern, especially on the web.  But wanted to call that out.